### DISCUSSION TITLE
Are pre-built executables available for Windows

### DISCUSSION THREAD

**Discussion by realtimedesigns** [rokath/trice#258] [Q&A] [answered]
Hello, Thank you for your efforts on this project.

The Trice user guide (Section 2.2) seems to imply that binaries exist but I could not find any in the repository.
Are prebuilt binaries (executables)  available as I did not really want to set up (and learn) the Go dev environment.
Best Regards
Ivan

**Comment by rokath** [comment, marked as answer]
https://github.com/rokath/trice/releases
